### PR TITLE
CORE-994: added some tests for tool listings.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/__tests__/tools.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+afterEach(() => {
+    mockAxios.reset();
+});
+
+test("Tool Table View renders", () => {
+    const component = renderer.create(<ToolListingTest />);
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```
Comment (psarando) [inline src/__tests__/tools.js:22]: I thought we wanted to move away from snapshots and use `chromatic` instead?
So in my last PR I simply rendered a component and unmounted it, to ensure it wouldn't crash in either phase.
```suggestion
    component.unmount();
```

Comment (slr71) [reply src/__tests__/tools.js:22]: I thought that matching the snapshot was using chromatic. 😕 I could easily be mistaken, though. I'm still a newbie when it comes to UI work.

Comment (psarando) [reply src/__tests__/tools.js:22]: I think chromatic automatically creates its own snapshots and rendered images based on the stories that are added in a PR.

My last PR #108 didn't include any snapshot files for the components I created, but chromatic still rendered all the stories I added.

Comment (sri2k1us) [reply src/__tests__/tools.js:22]: Chromatic automates snapshot creation and comparison. No need to manually create and update snapshot with each change 🎉

Comment (slr71) [reply src/__tests__/tools.js:22]: Ah, okay. Thanks. I'll update the code.

**Comment by slr71** [comment]
I think I've addressed everything, so I'm going to merge this now. Please feel free to ping me if there's anything I missed. I can fix any remaining problems in a subsequent pull request.

**Pull request merged**